.vscode/tasks.json

```diff
@@ -11,6 +11,12 @@
       "problemMatcher": [
         "$gcc"
       ],
+      "presentation": {
+        "echo": true,
+        "showReuseMessage": true,
+        "clear": false,
```

Comment:
"clear" : true works better for me by clearing output from previous run to avoid confusion